{{ cookiecutter.repo_name }}/.pre-commit-config.yaml

```diff
@@ -5,13 +5,17 @@ repos:
     rev: master
     hooks:
     -   id: flake8
-        args: ['src']
+        args: ['hooks', 'tests', '{{ cookiecutter.repo_name }}/src']
```

Comment:
```suggestion
        args: ['src']
```

Reply:
Should be resolved here. https://github.com/ukgovdatascience/govcookiecutter/pull/9/commits/28debd8b0eb80a00e98bcf75a8a75f3f052f3274